configure.ac

```
@@ -623,14 +625,18 @@ LT_INIT
 CFLAGS="$saved_CFLAGS"
 host_os=$old_host_os
 
+DEP_CC_PREFIX=''
 AS_CASE([$host],
   [sparc-sun-solaris*],
     [DEP_CC="false"],
   [*-pc-windows],
-    [AC_CHECK_TOOLS(
-      [DEP_CC],
-      [$DEP_CC gcc cc x86_64-w64-mingw32-gcc i686-w64-mingw32-gcc],
-      [false])],
+    [AS_IF([test x"$CC" = "xclang-cl"],
+      [DEP_CC="$CC"
+      DEP_CC_PREFIX='/clang:'],
```

```suggestion
    [AS_CASE(["$CC"],
      [clang-cl*],
        [DEP_CC="$CC"
        DEP_CC_PREFIX='/clang:'],
```
We could match on `clang-cl`, `clang-cl.exe` (especially if we're cross-compiling on a unix host, where there won't be a `.exe` extension). It's also possible that the executable is versioned with LLVM's version, as in `clang-cl-20` or `clang-cl-20.exe`. Maybe using the compiler vendor identifier is a better fit?

---

Oo - the vendor's a very good idea - you've made there be a concrete string in there for "clang-cl", right?

---

`msvc-*-clang-*`